robottelo/upgrade_utility.py

```diff
@@ -0,0 +1,150 @@
+"""Common Upgrade test utilities """
+
+from nailgun import entities
+from wait_for import wait_for
+from fabric.api import execute, run
+from robottelo.test import settings
+from upgrade.helpers.docker import docker_execute_command
+from robottelo.api.utils import call_entity_method_with_timeout
+
+
+class CommonUpgradeUtility(object):
+    """ CommonUpgradeUtility class for upgrade test cases """
+
+    def __init__(self, container_id=None, package=None):
+        self.client_container_id = container_id
+        self.package = package
+
+    def run_goferd(self):
+        """
+        Start the goferd process.
+        :return: status of execution
+        """
+
+        kwargs = {'async': True, 'host': settings.upgrade.docker_vm}
+        execute(
+            docker_execute_command,
+            self.client_container_id,
+            'pkill -f gofer',
+            **kwargs
+        )
+        execute(
+            docker_execute_command,
+            self.client_container_id,
+            'goferd -f',
+            **kwargs
+        )
+        status = execute(docker_execute_command, self.client_container_id, 'ps -aux',
```

Comment:
Can we keep `ps -ef` I have seen this https://github.com/SatelliteQE/robottelo/pull/7090 earlier